c/busybox-1.22.0/chroot-incomplete_false-no-overflow.c

```diff
@@ -333,10 +338,11 @@ static signed long int safe_write(signed int fd, const void *buf, unsigned long
 }
 
 // file include/libbb.h line 466
+extern int __VERIFIER_nondet_int(void);
 static void xchdir(const char *path)
 {
   signed int return_value_chdir$1;
-  return_value_chdir$1=chdir(path);
+  return_value_chdir$1=__VERIFIER_nondet_int();
```

Comment:
As above, let's please have an "implementation" of `chdir` (even when just returning a non-deterministic value).

Reply:
The problem with chdir is that it may require a different model in other benchmarks. Here is important only the return value, but if the benchmark would call `getcwd()` later, then this model is not sufficient.

Reply:
I would oppose modelling these functions to a specific benchmark. If more detailed modelling is required, we will add that, and `chroot` will benefit from it as well.